script/universal/MultisigBase.sol

```diff
@@ -3,7 +3,7 @@ pragma solidity ^0.8.15;
 
 import {console} from "forge-std/console.sol";
 import {IMulticall3} from "forge-std/interfaces/IMulticall3.sol";
-import {IGnosisSafe, Enum} from "@eth-optimism-bedrock/scripts/interfaces/IGnosisSafe.sol";
+import {IGnosisSafe, Enum} from "./IGnosisSafe.sol";
```

Comment:
Curious what if IGnosisSafe changes? We'll have to change it here ourselves as well?

Reply:
Our multisigs are already deployed and we control when they would upgrade. I think it actually makes more sense for this interface to be in our repo since it's our multisig interface.